### DISCUSSION TITLE
Possibility making default sort button renamable

### DISCUSSION THREAD

**Discussion by SoerenOhlsen** [microsoft-search/pnp-modern-search#2413] [Ideas]
Hi,

our customer mentioned to rename the default sorting-button. Maybe it can also be made configurable through the JSON configuration in the SortList section.

Or is there already a solution to do so? 

![image](https://user-images.githubusercontent.com/67585513/194125819-04bb52c5-3da1-4a00-b998-9de452aef0ed.png)

**Comment by wobba** [comment]
Do you want to rename to something custom or improve the German translation? The issue with providing a setting is that it only works for one language.

**Reply by SoerenOhlsen** [reply]
Yes, improve translation, maybe "Standardsortierung". That was the designation that our customer had suggested.

**Reply by wobba** [reply]
Changed in the next release.
